Plutarch/Api/V1.hs

```diff
@@ -254,30 +251,30 @@ deriving via
     (PConstant Plutus.PubKeyHash)
 
 newtype PPubKey (s :: S) = PPubKey (Term s PByteString)
-  deriving (PlutusType, PIsData) via (DerivePNewtype PPubKey PByteString)
+  deriving (PlutusType, PIsData, PEq, POrd) via (DerivePNewtype PPubKey PByteString)
 
-instance PUnsafeLiftDecl PPubKey where type PLifted PPubKey = PlutusCrpyto.PubKey
+instance PUnsafeLiftDecl PPubKey where type PLifted PPubKey = PlutusCrypto.PubKey
 deriving via
-  (DerivePConstantViaNewtype PlutusCrpyto.PubKey PPubKey PByteString)
+  (DerivePConstantViaNewtype PlutusCrypto.PubKey PPubKey PByteString)
   instance
-    (PConstant PlutusCrpyto.PubKey)
+    (PConstant PlutusCrypto.PubKey)
 
 newtype PSignature (s :: S) = PSignature (Term s PByteString)
-  deriving (PlutusType, PIsData) via (DerivePNewtype PSignature PByteString)
+  deriving (PlutusType, PIsData, PEq, POrd) via (DerivePNewtype PSignature PByteString)
 
-instance PUnsafeLiftDecl PSignature where type PLifted PSignature = PlutusCrpyto.Signature
+instance PUnsafeLiftDecl PSignature where type PLifted PSignature = PlutusCrypto.Signature
 deriving via
-  (DerivePConstantViaNewtype PlutusCrpyto.Signature PSignature PByteString)
+  (DerivePConstantViaNewtype PlutusCrypto.Signature PSignature PByteString)
   instance
-    (PConstant PlutusCrpyto.Signature)
+    (PConstant PlutusCrypto.Signature)
 
 ---------- Time
 
 newtype PPOSIXTime (s :: S)
   = PPOSIXTime (Term s PInteger)
   deriving (PIntegral) via (PInteger)
   deriving newtype (Num)
```

Comment:
Is this a valid instance? It's for `Num (PPOSIXTime s)` rather than `Num (Term s PPOSIXTime)`.

Reply:
It's valid, but it's certainly not useful..

Reply:
I wonder if `DerivePNewtype` should be able to derive `Num`, `Semigroup`, and `Monoid` then (over actual terms, of course). Would that be useful?

Reply:
It would definitely be.

Reply:
We should also add PlutusType instances.

Reply:
We have `PlutusType` and `PIsData` already, as well as `PEq` and `POrd`.